common/src/main/java/com/wynntils/models/stats/builders/MiscStatKind.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.stats.builders;
+
+import com.wynntils.models.stats.type.StatUnit;
+import java.util.Locale;
+
+public enum MiscStatKind {
+    // Health and mana
+    HEALTH("Health", StatUnit.RAW, "healthBonus"),
+    HEALTH_REGEN_PERCENT("Health Regen", StatUnit.PERCENT, "healthRegen"),
+    HEALTH_REGEN_RAW("Health Regen", StatUnit.RAW, "healthRegenRaw"),
+    LIFE_STEAL("Life Steal", StatUnit.PER_3_S, "lifeSteal"),
+    MANA_REGEN("Mana Regen", StatUnit.PER_5_S, "manaRegen"),
+    MANA_STEAL("Mana Steal", StatUnit.PER_3_S, "manaSteal"),
```

Comment:
I will have to ping @P0keDev for this. Did 2.0 make it so mana stats are not `/ x seconds`? Should we keep them this way?

Reply:
What do you mean? They are `PER_5_S` or `PER_3_S` meaning, "per 3 seconds" (i.e. "/3 s"), or 5. This is as Wynncraft is. I have not changed anything in this area..?

Except the old code had Athena rewrite both per 5 and per 3 s to a single "per 4 s", and then we had to re-split them in the mod again. That was just stupid, and nothing I kept. Is that what you are thinking about?

Reply:
> What do you mean? They are `PER_5_S` or `PER_3_S` meaning, "per 3 seconds" (i.e. "/3 s"), or 5. This is as Wynncraft is. I have not changed anything in this area..?
> 
> Except the old code had Athena rewrite both per 5 and per 3 s to a single "per 4 s", and then we had to re-split them in the mod again. That was just stupid, and nothing I kept. Is that what you are thinking about?

I think Wynn changed how mana works in 2.0. Maybe I am just so outdated in Wynn mechanics... Let's just leave this.

Reply:
The stats are still displayed the same way and do still represent the "per x seconds" value - what changed is how exactly the buffs are applied to the player. I.e., having 20/5s mana regen means you'll gain 4 mana per second